views/docs/show.hbs

```diff
@@ -1,6 +1,12 @@
-{{#each doc.sections}}
-  {{> doc
-    doc=this
-    currentLocale=../currentLocale
-  }}
-{{/each}}
+{{#if doc.html}}
+  <div class="sub-section position-relative" data-name="{{doc.name}}" data-lang="{{../currentLocale}}">
+    {{{doc.html}}}
+  </div>
+{{else}}
+  {{#each doc.sections}}
+    {{> doc
```

Comment:
Should we create an issue to remove dead code once https://github.com/electron/i18n/pull/1786 is merged?

I believe this template will not be used any more and we have the temporary fix as well.